de.tu_bs.cs.isf.e4cf.parts.project_explorer/project_explorer.e4xmi

```diff
@@ -4,7 +4,7 @@
   <fragments xsi:type="fragment:StringModelFragment" xmi:id="_RuDIkvfoEeiagocgPMLR0w" featurename="children" parentElementId="de.tu_bs.cs.isf.familymining.ppu_iec.rcp_e4.left_window" positionInList="index:0">
     <elements xsi:type="basic:Part" xmi:id="_RuDIk_foEeiagocgPMLR0w" elementId="de.tu_bs.cs.isf.e4cf.parts.project_explorer" contributionURI="bundleclass://de.tu_bs.cs.isf.e4cf.parts.project_explorer/de.tu_bs.cs.isf.e4cf.parts.project_explorer.controller.ProjectExplorerViewAdapter" label="Project Explorer" iconURI="platform:/plugin/de.tu_bs.cs.isf.e4cf.core/icons/Explorer_View/items/project_explore_24.png">
       <menus xsi:type="menu:PopupMenu" xmi:id="_RuDIlffoEeiagocgPMLR0w" elementId="de.tu_bs.cs.isf.e4cf.parts.project_explorer.context_menu" label="project_explorer_context_menu">
-        <children xsi:type="menu:DirectMenuItem" xmi:id="_RuDIlvfoEeiagocgPMLR0w" elementId="de.tu_bs.cs.isf.familymining.ppu_iec.rcp_e4.directmenuitem.new_folder" label="New Folder" iconURI="platform:/plugin/de.tu_bs.cs.isf.e4cf.core/icons/Explorer_View/items/project16.png" contributionURI="bundleclass://de.tu_bs.cs.isf.e4cf.parts.project_explorer/de.tu_bs.cs.isf.e4cf.parts.project_explorer.handlers.NewFolderHandler"/>
+        <children xsi:type="menu:DirectMenuItem" xmi:id="_RuDIlvfoEeiagocgPMLR0w" elementId="de.tu_bs.cs.isf.familymining.ppu_iec.rcp_e4.directmenuitem.new_folder" label="New Folder" iconURI="platform:/plugin/de.tu_bs.cs.isf.e4cf.core/icons/Explorer_View/items/folder16.png" contributionURI="bundleclass://de.tu_bs.cs.isf.e4cf.parts.project_explorer/de.tu_bs.cs.isf.e4cf.parts.project_explorer.handlers.NewFolderHandler"/>

```

Comment:
Icon inconsistency: the new folder wizard uses blue icon (project) but toolbar and now context menu that call the wizard use the yellow icon (directory)